OMCompiler/Compiler/BackEnd/CommonSubExpression.mo

```diff
@@ -545,7 +545,7 @@ algorithm
 
             // Save the rhs (call) as bind expression and set fixed=true
             var := BackendVariable.setBindExp(var, SOME(call));
-            var := BackendVariable.setVarFixed(var, true);
+            var := BackendVariable.makeParam(var);
```

Comment:
You could also just inline the function call here, don't know if it is really necessary.